SimPEG/electromagnetics/static/resistivity/simulation.py

```diff
@@ -489,7 +489,7 @@ def __init__(self, mesh, survey=None, bc_type="Robin", **kwargs):
         if mesh._meshType == "TREE":
             mesh.nodal_gradient
         elif mesh._meshType == "CYL":
-            bc_type == "Neumann"
+            bc_type = "Neumann"
```

Comment:
I assumed that we are intending to change the `bc_type` in this line. Correct me if I'm wrong.